scripts/integration/integration.py

```diff
@@ -486,14 +497,39 @@ def main():
         default='scripts/flamegraph',
         type=Path,
         help='Path to flamegraph script')
+    parser.add_argument(
+        '-v',
+        '--verbosity',
+        default='DEBUG',
+        help='Set the logging verbosity')
 
     args = parser.parse_args()
+
+    # Setup logging.
+    LOGGER.setLevel(logging.DEBUG)
+    fmt = "%(asctime)s %(levelname)-8s [%(name)s] %(message)s"
+    colored_formatter = coloredlogs.ColoredFormatter(fmt)
+    plain_formatter = logging.Formatter(fmt)
+    formatter = colored_formatter if sys.stdout.isatty() else plain_formatter
+    ch = logging.StreamHandler()
+    ch.setLevel(args.verbosity)
+    ch.setFormatter(formatter)
+    LOGGER.addHandler(ch)
+    class ShutdownHandler(logging.Handler):
+        def emit(self, record):
+            logging.shutdown()
+            sys.exit(1)
+    sh = ShutdownHandler(level=50)
+    sh.setFormatter(formatter)
+    LOGGER.addHandler(sh)
+
     if not args.set:
         args.set = Path(__file__).resolve().parent / 'default_set.yaml'
     args.set = args.set.resolve()
+    LOGGER.debug(f'resolved test set path to {args.set}')
 
     test_file = open(args.set, 'r')
-    test_dict = yaml.load(test_file)
+    test_dict = yaml.full_load(test_file)
```

Comment:
Interesting. Does this work around that annoying github warning?

Reply:
Yes, that's the issue. I don't know how github tracks this, though. If they go just by used version, then we need to do a bit more than this fix.

Reply:
I've added a new commit to update the version in the `requirements.txt`. This should fix the warning.